app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -96,6 +97,11 @@ class MainActivity : ThemedActivity() {
         SyncMessagesWorker.cancelWork(this)
     }
 
+    override fun onResume() {
+        super.onResume()
+        if (binding.drawerLayout.isOpen) colorSystemBarsWithMenuDrawer()
```

Comment:
Why from resume ?

Reply:
This only works in onResume, else the drawer is never detected as being "open"